main/helpers.py

```diff
@@ -33,3 +33,55 @@ def get_match_details(match):
         "player2_score": match.player2_score,
         "field": match.field.name
     }
+
+
+def create_new_match(p1_id, p2_id, p1_score, p2_score, field_id):
```

Comment:
I don't feel like this helper is useful - what was your rationale behind it?

Reply:
Yeah I created that because I was rerendering that view from one of the api endpoints, so I took the meat out of the view and put it in a helper. Later I found out redirecting was a much better solution, and didnt need to use the helper funciton more than once. I kept it because I thought it might make things simpler, but i think in the end it makes it more disjointed. I will put it back the way it was

Reply:
tldr: i will put it back the way it was

Reply:
looking back at this comment, I thought this was referencing a different helper method. This process is used twice in views.py, so I put it into a helper

Reply:
I see that it is used, but I'm not sure it helps to extract this. The entire function is a single statement, the function's name (`create_new_match`) is not more descriptive than the command itself (`Match.objects.create()`), so all it does is it saves you a couple of lines, but in return it obfuscates the arguments so now you have to know their order.
Anyway this is most probably a stylistic choice and it's up to personal taste, so you can leave it as is if you want to.